src/main/java/io/cdap/plugin/gcp/datastore/sink/util/DatastoreSinkConstants.java

```diff
@@ -34,7 +34,8 @@
   String CONFIG_PROJECT = "mapred.gcd.output.project";
   String CONFIG_USE_AUTOGENERATED_KEY = "mapred.gcd.output.use.autogenerated.key";
   String CONFIG_BATCH_SIZE = "mapred.gcd.output.batch.size";
-  String CONFIG_SERVICE_ACCOUNT_FILE_PATH = "mapred.gcd.output.service.account.file.path";
+  String CONFIG_SERVICE_ACCOUNT = "mapred.gcd.output.service.account";
+  String CONFIG_SERVICE_ACCOUNT_IS_FILE_PATH = "mapred.gcd.output.service.account.isfile";
```

Comment:
the string also needs to change to indicate that it's a file path. something like " mapred.gcd.output.service.account.isfilepath"